apps/src/templates/studioHomepages/SectionsTable.jsx

```diff
@@ -89,36 +100,56 @@ const SectionsTable = React.createClass({
         sectionCode: React.PropTypes.string.isRequired
       })
     ),
-    isRtl: React.PropTypes.bool.isRequired
+    isRtl: React.PropTypes.bool.isRequired,
+    isTeacher: PropTypes.bool.isRequired,
+  },
+
+  onLeave() {
+    console.log("How do I remove a student from a section?");
   },
```

Comment:
@tanyaparker and @poorvasingal are still working out details for this. TBD whose work item it is (me, @Bjvanminnen, someone else?) and whether only students with email accounts will have this privilege or not.